faucet-ui/src/utils/const.ts

```diff
@@ -0,0 +1,8 @@
+export enum Routes {
+  Home = "/",
+  Auth = "/auth",
+  Success = "/success",
+  Failure = "/failure",
+}
+
+export const ClientId = "58c63aeed6e97fd4929e"
```

Comment:
Is client id secret info? Or is it safe to expose?

Reply:
It is the ID of the application.
There's another `Client Secret`.

Reply:
OK cool.